build.sbt

```diff
@@ -14,22 +14,21 @@ lazy val commonSettings: Seq[Setting[_]]  = Seq(
   developers := List(
     Developer(id = "tenchi", name = "Hamza Haiken", email = "tenchi@team2xh.net", url = url("http://tenchi.me"))
   ),
-  licenses := Seq("MIT" -> url("https://github.com/Tenchi2xh/Scurses/blob/master/LICENSE")),
-  useGpg := true
```

Comment:
This has been depreciated because bouncy castle was removed in latest sbt-gpg so `useGpg` is always true now